test/e2e/memcached_test.go

```diff
@@ -623,26 +631,123 @@ func memcachedMetricsTest(t *testing.T, f *framework.Framework, ctx *framework.T
 	if err != nil {
 		return fmt.Errorf("could not get metrics Service: (%v)", err)
 	}
-
-	// Get operator pod
-	pods := v1.PodList{}
-	opts := client.InNamespace(namespace)
 	if len(s.Spec.Selector) == 0 {
 		return fmt.Errorf("no labels found in metrics Service")
 	}
 
-	for k, v := range s.Spec.Selector {
-		if err := opts.SetLabelSelector(fmt.Sprintf("%s=%s", k, v)); err != nil {
-			return fmt.Errorf("failed to set list label selector: (%v)", err)
+	response, err := getMetrics(t, f, s.Spec.Selector, namespace, "8383")
```

Comment:
Nit: Can we make the default portValues metricsPort `8383` and operatorMetricsPort and `8686` as constants in the `internal/pkg/scaffold/cmd.go` template?
That should make it easier to reference them here in the e2e tests.

Reply:
Same for Line 670

Reply:
Not sure we can reference them in the e2e tests as it's part of the internal package?

Reply:
I think the e2e tests in the SDK project can reference it's own internal pkg. We're already doing it:
https://github.com/operator-framework/operator-sdk/blob/master/test/e2e/memcached_test.go#L30-L33
https://github.com/operator-framework/operator-sdk/blob/master/internal/pkg/scaffold/constants.go
https://github.com/operator-framework/operator-sdk/blob/master/test/e2e/memcached_test.go#L226

With that we can define the constants as in the scaffold file for cmd.go as:
```Go
...

const (
	CmdFile = "main.go"
	MetricsPort = "8383"
	OperatorMetricsPort = "8686"
)

type Cmd struct {
	input.Input
	// Port to expose default controller-runtime metrics
	MetricsPort string
	// Port to expose operator specific kube-state style metrics
	OperatorMetricsPort string
}

func (s *Cmd) GetInput() (input.Input, error) {
	if s.Path == "" {
		s.Path = filepath.Join(ManagerDir, CmdFile)
	}
	if s.MetricsPort == "" {
		s.MetricsPort = MetricsPort
	}
	if s.OperatorMetricsPort == "" {
		s.OperatorMetricsPort = OperatorMetricsPort
	}
	s.TemplateBody = cmdTmpl
	return s.Input, nil
}

const cmdTmpl = `package main

import (
	"context"
	"flag"
...
)

// Change below variables to serve metrics on different host or port.
var (
    metricsHost = "0.0.0.0"
    metricsPort int32 = {{.MetricsPort}}
    operatorMetricsPort int32 = {{.OperatorMetricsPort}}
)
var log = logf.Log.WithName("cmd")
...
```

But not to hold up the PR, it's a minor issue which we can update later if you want to put a TODO.

@estroz Just double checking if I've outlined how to set the scaffold inputs via constants correctly above.

Reply:
That is correct. I agree this should be `const`-ized in a follow-up. I suggest adding a TODO.

Reply:
Added todo, will address in a follow up. Was not aware, thanks!